### ISSUE TITLE
Implicit default value set on non-nullable MoneyFields where none is expected

### ISSUE THREAD

**Issue by craiga** [django-money/django-money#411]
It seems that an unexpected implicit default value is being set on all non-nullable MoneyField values.

When adding the following field to an existing model:

```python
    tax_per_adult = MoneyField(_("price per adult"), decimal_places=5, max_digits=10)
```

…the following is output in Django's auto-generated migration:

```python
        migrations.AddField(
            model_name='flightsubbooking',
            name='tax_per_adult',
            field=djmoney.models.fields.MoneyField(decimal_places=5, default=Decimal('0.0'), max_digits=10, verbose_name='price per adult'),
        ),
```

Note that this field now has a default value of `Decimal('0.0')`.

I would expect that this field would behave like any other. That is, if no default value is specified and no explicit value is given, an error would be raised.

As this isn't happening, there's a risk of `tax_per_adult` silently being set to `0.0`.

I'm using django-money 0.13.2 and django 1.11.12.

**Comment by benjaoming** [comment]
That seems like a good idea.

Related: https://github.com/django-money/django-money/issues/145

**Comment by GeeWee** [comment]
The code that seems to set this default seems to be in `fields.py`
```python
# Line 191
    def setup_default(self, default, default_currency, nullable):
        if default is None and not nullable:
            # Backwards compatible fix for non-nullable fields
            default = 0.0
```
Not exactly sure what the comment means though. I assume that some of the other code might also depend on the amount never being `None` - so it might be a little more work than fixing this one line.

**Comment by Stranger6667** [comment]
The issue is fixed in #464, now there is no implicit `default` value.
